docs/providers/google.mdx

```diff
@@ -19,37 +19,38 @@ For more options and configurations, see the [Google Maps developer docs][2].
 ```js
 import { GoogleProvider } from 'leaflet-geosearch';
 
-const provider = new GoogleProvider({
-  params: {
-    key: '__YOUR_GOOGLE_KEY__',
-  },
-});
-
 // add to leaflet
 import { GeoSearchControl } from 'leaflet-geosearch';
 
-map.addControl(
-  new GeoSearchControl({
-    provider,
-    style: 'bar',
-  }),
-);
+const params = {
```

Comment:
@kstratis How do the changes to the docs here seem to you?

Reply:
That's perfectly fine!